src/Tutorials/Front-Ends-for-Orleans-Services.md

```diff
@@ -58,6 +58,7 @@ namespace WebApplication1
        	   ...
 ```
 
+The default configuration for the config has the port set to 40000, which your configuration file which we set up earlier in the documentation has been set to 30000
```

Comment:
I'm sorry, I'm a bit confused here. Maybe this could be rephrased in a more straightforward way? Or should we instead update the config file to match the default or maybe even not set the gateway port explicitly at all?